.pre-commit-config.yaml

```diff
@@ -0,0 +1,41 @@
+repos:
+-   repo: https://github.com/psf/black
+    rev: 19.3b0
+    hooks:
+    -   id: black
+        args: [--safe]
+        language_version: python3.7
+-   repo: https://github.com/asottile/blacken-docs
+    rev: v1.1.0
+    hooks:
+    -   id: blacken-docs
+        args: [--skip-errors]
+        additional_dependencies: [black==19.3b0]
+        language_version: python3.7
+-   repo: https://github.com/asottile/seed-isort-config
+    rev: v1.9.1
+    hooks:
+    -   id: seed-isort-config
+        args: [--application-directories, "src:."]
+-   repo: https://github.com/pre-commit/mirrors-isort
+    rev: v4.3.21
+    hooks:
+    -   id: isort
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.2.3
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: debug-statements
+    -   id: flake8
+        additional_dependencies: ["flake8-bugbear == 19.3.0"]
```

Comment:
this can be bumped to 19.8 :-) can you try to run pre-commit auto-update and see if anything else needs bumped?